test/app/adBlockTest.js

```diff
@@ -0,0 +1,27 @@
+/* global describe, it, before */
+
+const Brave = require('../lib/brave')
+const selectors = require('../lib/selectors')
+const messages = require('../../js/constants/messages')
+
+describe('adBlock', function () {
+  function * setup (client) {
+    yield client
+      .waitUntilWindowLoaded()
+      .waitForVisible('#window')
+      .waitForVisible(selectors.urlInput)
+  }
+
+  describe('blocks ruled ads', function () {
+    Brave.beforeAll(this)
+    before(function *() {
+      yield setup(this.app.client)
+    })
+
+    it('blocks a known rule on a local url', function *() {
+      yield this.app.client
+        .ipcSend(messages.SHORTCUT_NEW_FRAME, 'http://www.newsbtc.com/2016/02/14/bitcoin-price-weekly-analysis-primed-for-lift-off-2/')
+        .isExisting('.adfixedLeft').should.eventually.be.false
```

Comment:
@rob-blasetti just to expand on what @bbondy said, the main window and the webview in the tab are completely separate (they don't even run in the same process). You cannot access anything in the tab webview if you are attached to the main window and vice-versa. A single window with one tab will actually create two windows from webdriver's perspective. A window with two tabs will have 3 and two windows with one tab each will have 4